sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,153 @@
+# tag_based_pnp_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `tag_based_pnp_calibrator`. Although we provide pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../tag_based_pnp_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md)
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1gFrrchW9mWM1huWMYuJ0nWB2n1BfCJag).
+
+The provided rosbag includes four different topics `camera_info`, `image_rect_color/compressed`, `pointcloud_raw`, and `tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. During the calibration, please always make sure that tags are detected by both camera and lidar. The user could always check whether the tags appear in the `rviz` and the `image view` UI.
```

Comment:
> The user could always check whether the tags appear in the `rviz` and the `image view` UI.

This is not well written.
For example: The user can check the detection status of the tags by observing rviz and the `image view` UI. 

Here there should also be an explanation, because mostly rviz and the UI show the converged pairs.
Rviz will show the current lidartag detections with cyan frames but how about the image detections?

Reply:
Thanks!
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cc75c68d22709c28e96567be1b5d47320469fb34